lib/psi.rb

```diff
@@ -1850,14 +1850,12 @@ def processTBD(
     surface[:type] = :wall if typ.include?("wall")
 
     unless s.construction.empty?
-      construction = s.construction.get.to_Construction.get
+      construction = s.construction.get.to_ConstructionBase.get
```

Comment:
The issue here was that construction was an `OS:Construction:FfactorGroundFloor` object and not an `OS:Construction`, `PlanarSurface::construction` returns a `ConstructionBase`.  `ConstructionBase` doesn't have a `layers` method, I figured you didn't really need to check if `index < construction.layers.size` as long as `deratableLayer` wouldn't return an invalid layer (which it looks like it won't).

Reply:
@rd2 FYI

Reply:
@macumber: Thanks. Odd that this never popped up in previous testing.